lib/lighthouse/healthcare_cost_and_coverage/invoice/service.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'common/client/base'
+require 'lighthouse/healthcare_cost_and_coverage/configuration'
+require 'lighthouse/service_exception'
+
+module Lighthouse
+  module HealthcareCostAndCoverage
+    module Invoice
+      class Service < Common::Client::Base
+        configuration Lighthouse::HealthcareCostAndCoverage::Configuration
+        STATSD_KEY_PREFIX = 'api.lighthouse.hccc.invoice'
```

Comment:
doesnt look like this gets used, but if logging is going come later I'm cool with it

Reply:
Good call out. I believe we will use it in the future.